docs/src/content/docs/components/link-buttons.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Link Buttons
+description: Learn how to create link buttons in Starlight for visually distinct call-to-action links.
+---
+
+import { LinkButton } from '@astrojs/starlight/components';
+
+To display visually distinct call-to-action links, use the `<LinkButton>` component.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<LinkButton slot="preview" href="/getting-started/">
+	Read the docs
+</LinkButton>
+
+</Preview>
+
+## Import
+
+```tsx
+import { LinkButton } from '@astrojs/starlight/components';
+```
+
+## Usage
+
+Display a link button, useful for directing users to the most relevant or actionable content and often used on landing pages, using the `<LinkButton>` component.
```

Comment:
While it means we repeat ourselves slightly between the opening and this section, I think this still flows better so it’s worth doing:
```suggestion
Use the `<LinkButton>` component to display a visually distinct call-to-action link. A link button is useful for directing users to the most relevant or actionable content and is often used on landing pages
```